location/src/main/res/values/strings.xml

```diff
@@ -7,4 +7,6 @@
     <string name="locationSettingsLocationNeverAsked">Location Settings</string>
     <string name="locationSettingsLocationAllowed">Location Allowed</string>
     <string name="locationSettingsLocationDenied">Location Denied</string>
+    <string name="locationPromptNotNow">Now Right Now</string>
```

Comment:
We should change this to `Not Right Now`. I can do that when I'm adding the Spanish and Chinese translations for AIC-431 or similar.